qemu/tests/cfg/uefi_check_boot_option.cfg

```diff
@@ -0,0 +1,34 @@
+- uefi_check_boot_option:
+    only q35
+    only ovmf
+    type = uefi_check_boot_option
+    timeout = 360
+    start_vm = no
+    boot_menu = on
+    ovmf_log = "ovmf.log"
+    extra_params += " -global isa-debugcon.iobase=0x402 -debugcon file:%s"
```

Comment:
the debug log was added by default,  get output by vm.logsessions[\'seabios\'].get_output().
Why add it again?

Reply:
Because `vm.logsessions['seabios'].get_output()` can not get complete log info,so I add debug log to save complete log info.

Reply:
set start_vm = no, still can not get complete log? If then, this is OK.